src/legacy/ui/ui_render/bootstrap/startup.js.hbs

```diff
@@ -7,16 +7,41 @@ try {
   window.console.log(error);
 }
 
-var useBrowserColorScheme = window.localStorage.getItem('useBrowserColorScheme') === 'true' || false;
+/** @type {'browser' | 'light' | 'dark' | null} */
+var urlThemeParam = null;
+try {
+  urlThemeParam = new URLSearchParams(window.location.search).get('theme');
+  if (!urlThemeParam && window.location.hash.includes('?')) {
+    urlThemeParam = new URLSearchParams(window.location.hash.split('?')[1]).get('theme');
```

Comment:
we specify urls params in the url fragment?

Reply:
In a future enhancement, we MIGHT be having an interceptor on the server side to handle language and theme changes among others. After intercepting, it will clean the URL and redirect to the destination without the lang and theme. If we look into the hash here, we cannot unlook at it later and that would make the transition challenging if not impossble without a breaking change.

IMO, we should drop this piece and stick to the one above.